live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/ApplicationInfoProvider.java

```diff
@@ -59,7 +63,14 @@ public String getUserAgentData()
     private String calculateUserAgentData()
     {
         String applicationVersion = integrationProperties.application().version();
-        String repositoryVersion = discoveryApiClient.getRepositoryVersion();
+        String repositoryVersion = discoveryApiClient.getRepositoryVersion()
+                .orElseGet(() -> Optional.of(integrationProperties)
```

Comment:
I think it would make more sense to flip this logic. i.e. Use the value from the configuration in preference to the discovery API.

Reply:
Done.